### PR TITLE
IST-2058 - Add create volume mutation

### PR DESCRIPTION
**Pull request by deechauhan**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -58,6 +58,7 @@ jobs:
         uses: commitizen-tools/commitizen-action@master
         with:
           push: false
+          prerelease: beta
```
Comment (deechauhan) [inline .github/workflows/ci.yml:61]: We can bump the version to prerelease instead - much elegant solution (and I've disable pypi publish and release workflows)

File: move_ugc/gql_requests/volume.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    __typename
+"""
+
+expand_camera_settings = """
+    cameraSettings {
+        lens
+    }
+"""
+
+expand_sources_w_camera_settings = f"""
```
Comment (benmoveai) [inline move_ugc/gql_requests/volume.py:25]: I can see this being used for multicam and singlecam takes as well. Should this be variable be stored in a more central place. Doesn't have to be moved now., Just flagging for later

File: move_ugc/schemas/sources.py [hunk 3]
```diff
@@ -34,6 +34,39 @@ def _missing_(cls, key_value: object) -> Optional[str]:  # noqa: WPS120
...
 
 
+class CameraSettings(BaseModel):
+    """Camera settings input for creating a take."""
+
+    lens: str = Field(
+        description="Lens used for the take",
+        title="Lens",
+        examples=[
+            "goprohero10-2-7k",
```
Comment (benmoveai) [inline move_ugc/schemas/sources.py:44]: Should we link here to https://move-ai.github.io/move-ugc-api/getting-started/multicam/lenses/ which contains all the current valid lense?

File: move_ugc/schemas/volume.py [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+        title="Volume name",
+    )
+    area_type: AreaType = Field(
+        description="Area type of the volume.",
+        examples=["NORMAL", "LARGE"],
+        title="Area type",
+        alias="areaType",
+    )
+    human_height: float = Field(
+        description="Height of the human in the volume",
```
Comment (benmoveai) [inline move_ugc/schemas/volume.py:39]: Can this say `in metres`?

File: move_ugc/schemas/volume.py [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+    sources: Optional[List[Source]] = Field(
+        description="Sources associated with the volume",
+        examples=[
+            {
+                "deviceLabel": "my-device",
+                "file": {ID_LITERAL: "file-1fd863d5-875b-4e48-89bb-c6234e804738"},
+                "format": "MP4",
+            },
+            {
+                "deviceLabel": "my-device",
```
Comment (benmoveai) [inline move_ugc/schemas/volume.py:69]: Should this second device label be something different? i.e. we shouldn't have `my-device` twice in the examples

File: move_ugc/services/volume.py [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+class VolumeService(BaseService[HumanVolumeType]):
+    """Service which can be used to communicate with Volume type in UGC API.
+
+    To use this service, you need to instantiate it with a valid Move UGC client.
+
+    ```python
+    from move_ugc import MoveUgc
+    ugc = MoveUgc(api_key="my-api-key")
+
+    # Call take service methods directly
```
Comment (benmoveai) [inline move_ugc/services/volume.py:22]: This comment still uses the word `take`. This should say `volume`

File: tests/fixtures/conftest/volume.py [hunk 7]
```diff
@@ -0,0 +1,269 @@
...
+    """Fixture to return a fake volume.
+
+    Args:
+        volume_fixtures_path (str): Path to volume fixtures.
+        metadata_for_update (dict[str, str]): Metadata for update.
+
+    Returns:
+        FakeVolumeJson: Fake volume.
+    """
+    with open(f"{volume_fixtures_path}/fake_volume.json") as volume_json:
```
Comment (benmoveai) [inline tests/fixtures/conftest/volume.py:30]: Can we use `os.join` here. I think having the `/` in will cause the tests to not run on windows - where they could be run potentially in the future.

File: move_ugc/gql_requests/volume.py (hunk 2 above)
Comment (deechauhan) [reply move_ugc/gql_requests/volume.py:25]: Yes - possibly camera_settings.py - I'll create a ticket for this

Comment (deechauhan) [reply move_ugc/gql_requests/volume.py:25]: https://moveai.atlassian.net/browse/IST-2247

File: move_ugc/schemas/sources.py (hunk 3 above)
Comment (deechauhan) [reply move_ugc/schemas/sources.py:44]: Yep - that's a good idea - I'll add this

File: move_ugc/gql_requests/volume.py (hunk 2 above)
Comment (benmoveai) [reply move_ugc/gql_requests/volume.py:25]: It's not just camera settings - but `expand_sources_w_camera_settings` as well. As sources on a multicam take can have camera settings (and possibly on a singlecam take in the future)

Comment (deechauhan) [reply move_ugc/gql_requests/volume.py:25]: Yes definitely, we can move it to a central place (its own graphql file)  - as part of the ticket I just created?

File: move_ugc/schemas/volume.py [hunk 8]
```diff
@@ -0,0 +1,81 @@
...
+        title="Volume name",
+    )
+    area_type: AreaType = Field(
+        description="Area type of the volume.",
+        examples=["NORMAL", "LARGE"],
+        title="Area type",
+        alias="areaType",
+    )
+    human_height: float = Field(
+        description="Height of the human in meters",
```
Comment (benmoveai) [inline move_ugc/schemas/volume.py:39]: Can we use the UK spelling (`metres`)?

**Review by benmoveai — APPROVED** [review]

File: move_ugc/schemas/volume.py (hunk 8 above)
Comment (deechauhan) [reply move_ugc/schemas/volume.py]: Done - can you reapprove?

**Pull request merged**